tesseract_python/tesseract_python/swig/eigen_types.i

```diff
@@ -24,21 +24,37 @@
  * limitations under the License.
  */
 
+// workaround Eigen::Ref argout
+
+namespace Eigen
+{
+  template <typename T> class Ref;
+  class MatrixXd;
+}
+
+%feature("valuewrapper") Eigen::Ref<Eigen::MatrixXd>;
+%template() Eigen::Ref<Eigen::MatrixXd>;
+
+
 %eigen_typemaps(Eigen::Vector2d);
 %eigen_typemaps(Eigen::Vector3d);
 %eigen_typemaps(Eigen::Vector4d);
-%eigen_typemaps(Eigen::Isometry3d);
+//%eigen_typemaps(Eigen::Isometry3d);
```

Comment:
Should this be commented out?

Reply:
Yeah, that typemap is defined in a different file now. I'll delete the line to make it less confusing.

Reply:
I removed that line